dp/cloud/python/magma/configuration_controller/response_processor/strategies/response_processing.py

```diff
@@ -370,20 +362,20 @@ def _update_grant_from_response(response: DBResponse, grant: DBGrant) -> None:
 
 
 def _terminate_all_grants_from_response(response: DBResponse, session: Session) -> None:
-    cbsd_id = response.payload.get(
-        CBSD_ID,
-    ) or response.request.payload.get(CBSD_ID)
+    cbsd_id = response.cbsd_id
```

Comment:
Is there always `cbsd_id` in a response, in other words is it possible that `cbsd_id` is only in a request, but not in a response?